main/webapp/modules/core/MOD-INF/controller.js

```diff
@@ -245,8 +247,8 @@ function registerImporting() {
   IM.registerExtension(".json", "text/json");
   IM.registerExtension(".js", "text/json");
 
-  IM.registerExtension(".xls", "binary/text/xml/xls/xlsx");
-  IM.registerExtension(".xlsx", "binary/text/xml/xls/xlsx");
+  IM.registerExtension(".xls", "application/vnd.ms-excel");
+  IM.registerExtension(".xlsx", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet");
```

Comment:
This should be kept: here, "binary/text/xml/xls/xlsx" serves as an internal identifier for the format, and does not need to be an official MIME type. The correct place to add the mapping is further down, with `IM.registerMimeType`.